eZ/Publish/Core/Repository/Tests/Service/Mock/PermissionsCriterionHandlerTest.php

```diff
@@ -20,19 +19,23 @@
  */
 class PermissionsCriterionHandlerTest extends BaseServiceMockTest
 {
-    protected $repositoryMock;
-
     /**
      * Test for the __construct() method.
      */
     public function testConstructor()
     {
-        $repositoryMock = $this->getRepositoryMock();
-        $handler = new PermissionsCriterionHandler($repositoryMock);
+        $permissionResolverMock = $this->getPermissionResolverMock();
+        $limitationServiceMock = $this->getLimitationServiceMock();
+        $handler = $this->getPermissionsCriterionHandlerMock();
 
         $this->assertAttributeSame(
-            $repositoryMock,
-            'repository',
+            $permissionResolverMock,
+            'PermissionResolver',
```

Comment:
@pspanja: Travis fails here - also should be `permissionResolver`. I missed that when looking for occurences of this.